### PR TITLE
update to prod

### PR DESCRIPTION
**Pull request by Timtech4u**
@kerry-okpere  please check comments.  @findtoni can merge after review.

### CODE REVIEW

File: src/pages/legal/Terms.vue [hunk 1]
```diff
@@ -0,0 +1,165 @@
+<template>
+  <Layout>
+    <PageHero color="#3C87D1" title="Term of Service" align="center" display="none" adisplay="none" />
+    <div class="container terms">
+      <div class="row no-gutters">
+        <div class="col-md-7 terms__text ">
+          <p><i>Effective on May 31, 2020</i></p>
+          <p> Mercuriemart is the best and easiest way to create your very own ecommerce store.
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:8]: @kerry-okpere  Please update to Seller

File: src/components/NavHeader/index.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
                 <div class="col col-xl-8">
                     <nav class="main d-none d-md-block">
                         <a href="#" class="pr">Why Mercuriemart?</a>
```
Comment (Timtech4u) [inline src/components/NavHeader/index.vue:12]: @kerry-okpere  Please update to Seller

Comment (Timtech4u) [inline src/components/NavHeader/index.vue:12]: @kerry-okpere  Please update to Seller

File: src/components/NavHeaderInner/index.vue [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
                     <g-link exact to="/">
-                        <img v-scroll-class:show=100 class="hide" src="../../assets/logo/logo.svg" width="120" alt="Mercuriemart">
-                        <img v-scroll-class:hide=100 src="../../assets/logo/logo-white.svg" width="120" alt="Mercuriemart">
+                        <img v-scroll-class:show=100 class="hide" src="../../assets/seller/LogoBlue.svg" width="120" alt="Seller">
+                        <img v-scroll-class:hide=100 src="../../assets/seller/LogoWhite.svg" width="120" alt="Seller">
                     </g-link>
                 </div>
                 <div class="col col-xl-8">
                     <nav class="main d-none d-md-block">
                         <a href="#" class="pr">Why Mercuriemart?</a>
```
Comment (Timtech4u) [inline src/components/NavHeaderInner/index.vue:13]: @kerry-okpere  Please update to Seller

File: src/pages/legal/Terms.vue [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            <li><a href="#account-activation" v-smooth-scroll="{offset: -40}">Account Activation</a></li>
+            <li><a href="#payment" v-smooth-scroll="{offset: -40}">Payment Methods &amp; Payment Accounts</a></li>
+            <li><a href="#sendbox" v-smooth-scroll="{offset: -40}">Sendbox</a></li>
+          </ol>
+        </div>
+        <div class="row">
+          <div class=" terms__content">
+            <h3  id="introduction">1. Introduction</h3>
+            <p>
+              By signing up for a Mercuriemart Account (as defined in <a href="#account-terms"><span>Section 2</span></a>) or by using any Mercuriemart Services (as defined below), you are agreeing to be bound by the following terms and conditions (the “Terms of Service”).
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:26]: @kerry-okpere  Please update to Seller

File: src/pages/legal/Terms.vue [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+            </p>
+            <p>
+              As used in these Terms of Service, “we”, “us” and “Mercuriemart” means the applicable Mercuriemart Contracting Party (as defined in <a href="#sendbox"><span>Section 5</span></a> below).
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:29]: @kerry-okpere  Please update to Seller

File: src/pages/legal/Terms.vue [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+            </p>
+            <p>
+              The services offered by Mercuriemart under the Terms of Service include various products and services to help you sell goods and services to buyers online.
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:32]: @kerry-okpere  Please update to Seller

File: src/pages/legal/Terms.vue [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+               Any new features or tools which are added to the current Services shall be also subject to the Terms of Service. 
+               You can review the current version of the Terms of Service at any time at  <a href="https://www.Mercuriemart.com/privacy/terms-of-service"><span>https://www.Mercuriemart.com/legal/terms</span></a>.
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:34]: @kerry-okpere  Please update to https://www.seller.com/privacy/terms-of-service

File: src/pages/legal/Terms.vue [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+            <ol>
+              <li>
+                <p>
+                  To access and use the Services, you must register for a Mercuriemart account (“Account”) by providing your full legal name, current address, phone number, a valid email address, and any other information indicated as required. 
+                  Mercuriemart may reject your application for an Account, or cancel an existing Account, for any reason, in our sole discretion.
+                </p>
+              </li>
+              <li>
+                <p>
+                  You must be the older of: (i) 18 years, or (ii) at least the age of majority in the jurisdiction where you reside and from which you use the Services to open an Account.
```
Comment (Timtech4u) [inline src/pages/legal/Terms.vue:53]: @kerry-okpere  so much Mercuriemarts :D

File: src/components/NavHeaderInner/index.vue (hunk 3 above)
Comment (abcubed3) [reply src/components/NavHeaderInner/index.vue:13]: Can we increase the size of the logo here and at the bottom. Since it doesn't look bold enough

**Pull request merged**